src/math/warpExposure.cc

```diff
@@ -160,11 +160,11 @@ std::string NearestWarpingKernel::NearestFunction1::toString(std::string const &
 
 std::shared_ptr<SeparableKernel> makeWarpingKernel(std::string name) {
     typedef std::shared_ptr<SeparableKernel> KernelPtr;
-    boost::cmatch matches;
-    static const boost::regex LanczosRE("lanczos(\\d+)");
+    std::cmatch matches;
+    static const std::regex LanczosRE("lanczos(\\d+)");
     if (name == "bilinear") {
         return KernelPtr(new BilinearWarpingKernel());
-    } else if (boost::regex_match(name.c_str(), matches, LanczosRE)) {
+    } else if (std::regex_match(name.c_str(), matches, LanczosRE)) {
```

Comment:
Can the `c_str()` here and the later `istringstream` be dropped since the pattern is so simple?